scripts/auto-rebase/last_lvms_rebase.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash -x
-./scripts/auto-rebase/rebase-lvms.sh to "registry.redhat.io/lvms4/lvms-operator-bundle:v4.17.0-43"
+./scripts/auto-rebase/rebase-lvms.sh to "registry.redhat.io/lvms4/lvms-operator-bundle:v4.18.2-2"
```

Comment:
This file is changed as part of running `rebase-lvms.sh`. If you run the command you changed (the new one) and add all changes to this PR that should be fine.